app/controllers/entry_controller.rb

```diff
@@ -4,16 +4,14 @@ class EntryController < ApplicationController
   before_filter :load_entry, :only => [:comment]
 
   def comment
-    begin
-      if @challenge.participator?(current_user) && params[:comment] && params[:comment][:text].size > 0
-        @entry.comments.push Comment.new(:comment => params[:comment][:text], :nickname => current_user.nickname)
-        @challenge.save
-      end
-
-    rescue Exception => e
-    ensure
-      redirect_to challenge_path(params[:challenge])
+    if @challenge.participator?(current_user) && params[:comment] && params[:comment][:text].size > 0
+      @entry.comments.push Comment.new(:comment => params[:comment][:text], :nickname => current_user.nickname)
+      @challenge.save
     end
+  rescue Exception
```

Comment:
Throughout this controller we are rescuing Exception, not sure this is the best strategy. If we are going to use such a large scope to rescue would it not be better to rescue StandardError?